asyncapi/quickbooks/Asyncapi.yaml

```diff
@@ -0,0 +1,74 @@
+asyncapi: "2.4.0"
+x-ballerina-event-identifier:
+  type : "body"
+  path : "eventNotifications.dataChangeEvent."
+info:
+  title: QuickBook Webhooks
+  version: "1.0.0"
+  description: |
+    Use webhooks to receive event triggered callbacks for entities that your app needs to stay on top of.
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+defaultContentType: application/json
+
+externalDocs:
+    description: Common properties returned from Event
+    url: "https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks"
+
+channels:
+  app:
+    subscribe:
+      message:
+        oneOf:
+          - x-ballerina-event-type: "Account"
+            externalDocs:
+              description: Webhook docs
+              url: "https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks"
+            payload:
+              $ref: "#/components/schemas/quickBookEvent"
+            summary: Subscribe to only the message events that mention your app
```

Comment:
Is this added by us? or we copied from quickbook docs?

Shouldn't this be like 
`Subscribe only to the message events that mention your app`